File: comms/push-notifications/apps/update-app/update-app.rb
require "twiliocomms"

client = TwilioComms::Client.new(
  account_id: "TWILIO_ACCOUNT_SID",
  auth_token: "TWILIO_AUTH_TOKEN"
)

client.push_notifications.apps.update(
  app_name: "appName",
  is_default: true
)

File: comms/push-notifications/apps/update-app/update-app.php
<?php

namespace Example;

use Twilio\Comms\TwilioComms;
use Twilio\Comms\PushNotifications\Apps\Requests\AppsUpdateRequest;

$client = new TwilioComms(
    accountId: 'TWILIO_ACCOUNT_SID',
    authToken: 'TWILIO_AUTH_TOKEN',
);
$client->pushNotifications->apps->update(
    'appName',
    new AppsUpdateRequest([
        'isDefault' => true,
    ]),
);
